src/activity-window-port.js

```diff
@@ -417,28 +403,17 @@ class ActivityWindowRedirectPort {
     return ActivityMode.REDIRECT;
   }
 
-  /** @override */
-  getTargetOrigin() {
-    return this.targetOrigin_;
-  }
```

Comment:
Mainly to disambiguate when the parameters should be checked. If used naively, currently most of code stumbles on "cancel" signals thinking them to be insecure. Otherwise it sort of seems duplicate data between port and result. If desired - I can always add later. That'd be a non-breaking change in the future.